src/components/VonayeProgress.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import VoyageProgress from './VoyageProgress'
+
+test('render port names', () => {
+    render(<VoyageProgress   
+        portOfLoading="Copenhagen" 
+        portOfDischarge="Oslo"
+        departureTime="Mon, Jul 8 2022 10:00"
+        arrivalTime="Mon, Jul 8 2022 18:00"/>
+        )
+    const portOfLoading = screen.getByText('Copenhagen')
+    const portOfDischarge = screen.getByText('Oslo')
+    expect(portOfLoading && portOfDischarge).toBeInTheDocument()
+  });
+
+test('render correct number of blue dots at a given progress', () => {
```

Comment:
Is it blue dots or dark dots?

Reply:
The test is like a story. It should not say what it does but why that happens. If I create a roman number converter and have tests like 'should return 333 for CCCXXXIII' and 'should return 87 for LXXXVII' that won't help the reader to figure out whats going on. What is the correct number? I have to check the test in order to figure out that every second progress creates a new dot. I'd state this inside the test name.

Reply:
dark blue dots more precisely

Reply:
There is a concept of data providers although I am not a big fan of it. The point is to test the edge or corner cases. What if the progress is 0, what if its 10, what if its in the middle?

Reply:
how does this look? 

```
it('render 10 dark blue dots when progress is 100%', () => {
    const { container } = render(<VoyageProgress   
        portOfLoading="Copenhagen" 
        portOfDischarge="Oslo"
        departureTime="Sat, Oct 1 2022 09:00"
        arrivalTime="Sat, Oct 1 2022 17:00"/>
        )
    const progress = 100 / 10
    const darkDots = container.getElementsByClassName("dark-dot");
    // needs to be divided by 2 because componnet renders twice
    expect(darkDots.length / 2).toBe(progress);
  }); 
```